### DISCUSSION TITLE
synching predictive settings to nightscout

### DISCUSSION THREAD

**Discussion by mwarrildt** [NightscoutFoundation/xDrip#3573] [Q&A]
I use both xDrip+ and Nightscout and have a question about the data they share/sync.

xDrip+ is my 'Point of Contact' with the data - both for viewing and entering.  Nightscout is my repository, which I share with my endocrinologist.  

As currently configured, xDrip+ syncs bg and treatments to Nightscout but not the basal program and carb ratios and correction factors that I'm using.  AFAIK, I must do that manually. (or maybe I don't?)

It would be convenient (i.e., 'nice to have') if these were also synced to Nightscout so that my doctor can be confident that the data he's seeing is current.  I understand that xDrip+ is complicated and there are many important issues and requests on the table.  I also recognize that there are downstream ramifications to be considered.  That said, I'd like to offer this for consideration.

Thanks
Mark

**Comment by Navid200** [comment]
Can you be specific?
Can you identify exactly what parameter(s) you want to be synced with Nightscout?

**Reply by mwarrildt** [reply]
Sure.
basal profile, carb ratios, & insulin sensitivity - along with their associated time segments

**Reply by Navid200** [reply]
Do you use AAPS?

**Reply by mwarrildt** [reply]
I do not.
